csunplugged/templates/plugging_it_in/lesson.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base-students.html" %}
+
+{% load i18n %}
+{% load static %}
+{% load render_html_field %}
+{% load django_bootstrap_breadcrumbs %}
+
+{% block title %}
+  {{ topic.name }} Programming Exercises
+{% endblock title %}
+
+{% block breadcrumbs %}
+  {% breadcrumb topic.name "plugging_it_in:topic" topic.slug %}
+  {% breadcrumb lesson.name "plugging_it_in:lesson" topic.slug lesson.slug %}
+{% endblock breadcrumbs %}
+
+{% block page_heading %}
+  <h1 id="{{ lesson.slug }}">{{ lesson.name }} programming exercises</h1>
```

Comment:
```suggestion
  <h1 id="{{ lesson.slug }}">{% blocktrans trimmed %}{{ lesson.name }} programming exercises{% endblocktrans %}</h1>
```

Reply:
See above comment

Reply:
This will (should?) have the same problem you encountered before. Basically anything accessing object attributes will need to use the with statement inside the blocktrans tag.